src/collision.cpp

```diff
@@ -614,3 +489,212 @@ collisionMoveResult collisionMoveSimple(Environment *env, IGameDef *gamedef,
 
 	return result;
 }
+
+bool collisionCheckIntersection(Environment *env, IGameDef *gamedef,
+		const aabb3f &box_0, const v3f &pos_f, ActiveObject *self,
+		bool collideWithObjects)
+{
+	#define PROFILER_NAME(text) (s_env ? ("Server: " text) : ("Client: " text))
+	Map *map = &env->getMap();
+	ServerEnvironment *s_env = dynamic_cast<ServerEnvironment*>(env);
+
+	ScopeProfiler sp(g_profiler, PROFILER_NAME("collisionInCollision()"), SPT_AVG);
+
+	/*
+		Collect node boxes
+	*/
```

Comment:
unhelpful comment (also misleading since there isn't a similar header comment for the collection of object boxes)

Reply:
There are no similar header comment fox object boxes in `collisionMoveSimple`, should I fix it somehow?

Reply:
Fixed also in `collisionMoveSimple`